README.md

```diff
@@ -1,7 +1,7 @@
 ![Valyria Tear](https://raw.githubusercontent.com/ValyriaTear/ValyriaTear/master/data/boot_menu/valyria_logo.png)
 
-**Development blog:** http://valyriatear.blogspot.com  
```

Comment:
As I told @RyDroid not so long ago, adding two trailing space in a md file forces newlines. So here, you've broken the files by removing them.
Please do not touch trailing spaces in md files. Could you fix this?